docs/src/atoms/dropdown.js

```diff
@@ -1,15 +1,25 @@
+/* eslint-disable */
+
+/**
+ * NOTE: The props 'button' part of this component's documentation has been is deprecated as of Feb. 11, 2020.
+ * Please use the DropdownButton component instead.
+ * We are disabling Eslint for this file because of the above note and wanting to
+ * deprecate it fully in hopes to rip out the select portion of it as well.
```

Comment:
Can I better understand why it is necessary to completely disable ESLINT?

Reply:
Because I don't see a need to fix the whole entire file when it's about to be fully deprecated.